src/types/promotions.d.ts

```diff
@@ -121,6 +121,7 @@ export interface ItemPercentDiscountSchema {
   targets: string[] | 'all'
   percent: number
   target_conditions?: Conditions
+  target_attributes?: PromotionAttribute
```

Comment:
Should this be an array per the [API documentation](https://documentation.elasticpath.com/commerce-cloud/docs/api/promotions/create-item-percent-discount-promotion.html#the-schema-object)? 
```suggestion
  target_attributes?: PromotionAttribute[]
```

Reply:
yes!